service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2408,6 +2480,25 @@ components:
         round-trip:
           type: string
 
+    ExportPreviewRequest:
+      type: object
+      properties:
+        study:
+          type: string
+          nullable: false
```

Comment:
I think this is non-nullable by default and instead it should be made non-optional by putting it in the `required:` list. As far as I can tell, OpenAPI differentiates between nullable and optional, though I don't know how that maps to the generated Java code.